etna/transforms/timestamp/fourier.py

```diff
@@ -132,6 +179,36 @@ def _construct_answer(df: pd.DataFrame, features: pd.DataFrame) -> pd.DataFrame:
         result.columns.names = ["segment", "feature"]
         return result
 
+    # TODO: добавить обработку пропусков
```

Comment:
Should we do something here?

Reply:
No, we shouldn't do anything, functions that used to compute features are dealing with NaNs fine (NaN -> NaN).